testdata/p4_16_samples_outputs/psa-table-hit-miss.p4-stderr

```diff
@@ -0,0 +1,3 @@
+psa-table-hit-miss.p4(62): [--Wwarn=invalid_header] warning: accessing a field of a potentially invalid header hdr.ethernet
+            hdr.ethernet.srcAddr : exact;
```

Comment:
I am not sure that this message is desirable. Why does it appear here?
Because you are assuming the fields of hdr may be invalid?

Reply:
If I am not wrong, after the first if statement in that control, the header will be invalid after the then branch is executed, or will be valid if the condition in the if is false. So for the second apply call, the header might be invalid.
But I am considering removing the line that gives warnings about them for now (until I test the potentially invalid header analysis thoroughly).